.travis.yml

```diff
@@ -25,7 +25,7 @@ jobs:
               - set -e
               - if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then export COMPONENT_TAG_EXTENSION="-PR${TRAVIS_PULL_REQUEST}-${TRAVIS_COMMIT}"; fi;
               - make component/build
-#              - if [ "$DOCKER_PASS" != "" ]; then make component/push ; else echo "No DOCKER_PASS. Skipping image push."; fi
+              - if [ "$DOCKER_PASS" != "" ]; then make component/push ; else echo "No DOCKER_PASS. Skipping image push."; fi
```

Comment:
Need to comment this out.  prow is pushing the image now

Reply:
yeah, prow was broken so i could not get to any image :(